requirements/dev.txt

```diff
@@ -4,6 +4,10 @@
 #
 #    pip-compile --allow-unsafe requirements/dev.in
 #
+asttokens==2.1.0
+    # via stack-data
+backcall==0.2.0
+    # via ipython
```

Comment:
The cookiecutter changed some of the `requirements/*.in` files so I re-ran `make requirements` to update the `*.txt`'s